### DISCUSSION TITLE
How can I use the function 'pinocchio.getFrameJacobian' in python

### DISCUSSION THREAD

**Discussion by mzq740556549** [stack-of-tasks/pinocchio#2009] [Q&A] [answered]
I saw an function named pinocchio::getFrameJacobian in c++ , but now I want to use it in python . Do you have an example about it ? Thank you !

**Comment by jcarpent** [comment]
Here is a tiny example of how to extract the documentation using iPython:
```python
import puPython 3.10.10 | packaged by conda-forge | (main, Mar 24 2023, 20:17:34) [Clang 14.0.6 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.13.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pinocchio as pin
In [2]: pin.getFrameJacobian?
Docstring:
getFrameJacobian( (Model)model, (Data)data, (int)frame_id, (ReferenceFrame)reference_frame) -> numpy.ndarray :
    Computes the Jacobian of the frame given by its ID either in the local or the world frames.
    The columns of the Jacobian are expressed in the LOCAL frame coordinates system.
    In other words, the velocity of the frame vF expressed in the local coordinate is given by J*v,where v is the joint velocity.
    computeJointJacobians(model,data,q) and updateFramePlacements(model,data) must have been called first.
Type:      function
```

**Reply by jcarpent** [reply, marked as answer]
You can just call:
```python
pin.computeJointJacobians(model,data,q)
pin.getFrameJacobian(model,data,model.getFrameId("frame_name"),pin.LOCAL)
```
I hope this will help you.

**Reply by mzq740556549** [reply]
I will try later. Thank you!
